test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/ControllerTest.cs

```diff
@@ -153,6 +177,27 @@ public void Controller_View_WithParameterViewNameAndViewModel_SetsResultViewName
         }
 
         [Fact]
+        public void Controller_View_WithParameterNullViewModel_MaintainsNullModel()
+        {
+            // Arrange
+            var controller = new TestableController()
+            {
+                ViewData = new ViewDataDictionary(new EmptyModelMetadataProvider()),
+                TempData = new TempDataDictionary(new DefaultHttpContext(), Mock.Of<ITempDataProvider>()),
+            };
+
+            // Act
+            var actualViewResult = controller.View(model: null);
```

Comment:
Should also test:
- `controller.PartialView(model: null)`

Less sure whether it's worth confirming the other overloads:
- `controller.View(viewName: "something", model: null)`
- `controller.PartialView(viewName: "something", model: null)`